website/blog/2018-09-13-decorators.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title:  "TC39 Standards-track Decorators in Babel"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2018-09-13 23:00:00
+categories: announcements
+share_text: "TC39 Standards-track Decorators in Babel"
+---
+
+Babel 7.1.0 finally supports the new decorators proposal: you can try it out by using the `@babel/plugin-proposal-decorators` plugin 🎉.
+
+<!--truncate-->
+
+## A bit of history
+
+Decorators were [first proposed](https://github.com/wycats/javascript-decorators/blob/696232bbd997618d603d6577848d635872f25c43/README.md) by [Yehuda Kats](https://github.com/wycats) more than three years ago. TypeScript released support for decorators in [version 1.5](https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#typescript-15) (2015) alongside with many ES6 features: modules, destructuring, `let`/`const`, `for ... of` and computed properties.
+Some major frameworks, like Angular and MobX, started using them to enhance their developer experience: this made decorators popular and gave the community a false sense of stability.
+
+Babel first implemented decorators in [version 5](https://github.com/babel/babel/blob/master/.github/CHANGELOG-v5.md#500), but they had been removed in Babel 6 because the proposal was still in flux. Logan Smyth created an unofficial plugin ([`babel-plugin-transform-decorators-legacy`](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy)) which replicated the Babel 5 behavior; it has then been moved to the official Babel repository during the first Babel 7 alpha release. This plugin still used the old decorators semantics, because it wasn't clear yet what the new proposal would have been.
+
+Since then, after that [Daniel Ehrenberg](https://github.com/littledan) and [Brian Terlson](https://github.com/bterlson) become co-authors of the proposal along with [Yehuda Katz](https://github.com/wycats), it has been almost completely rewritten. Not everything has been decided yet, and there isn't a compliant implementation as of today.
```

Comment:
Can drop "after that":

> Since then, [Daniel Ehrenberg](https://github.com/littledan) and [Brian Terlson](https://github.com/bterlson) joined Yehuda as co-authors of the proposal and nearly all of it was rewritten.